docs/tutorials/storage.rst

```diff
@@ -1,249 +1,261 @@
-
-The Python programming language contains easy to use and powerful ways in which
-to work with a computer's file system. MicroPython on the micro:bit implements
-a useful subset of these features to make it easy to read and write files on
-the device, while also providing consistency with other versions of Python.
+[W języku angielskim słowo file, tłumaczone jako
+plik, oznacza także teczkę na dokumenty - przyp.tłum.]
+
+Plik (komputerowy) jest, posiadającym nazwę, zasobem cyfrowym, który
+przechowywany jest w systemie plików. Zasoby takie przechowują użyteczne
+informacje jako dane. To swego rodzaju, posiadający nazwę, pojemnik
+przechowujący informacje. Zazwyczaj nazwa pliku oddaje to co plik zawiera.
+Zwyczajowo nazwy plików komputerowych kończy się sufiksem ``.coś``. Zazwyczaj
+to ``.coś`` wskazuje jaki typ danych został użyty do reprezentacji informacji.
+Na przykład, ``.txt`` oznacza plik tekstowy, ``.jpg`` obraz w formacie JPEG
+a ``.mp3`` dane dźwiękowe przetworzone do postaci cyfrowej algorytmem MP3.
+
+Niektóre systemy plików (taki jak ten na Twoim laptopie
+albo komputerze stacjonarnym) pozwalają na organizowanie plików
+w katalogi - posiadające nazwę pojemniki, które grupują powiązane ze sobą
+pliki i podkatalogi. Jednakże *system plików dostępny w MicroPythonie
+jest płaskim systemem plików*. Płaski system plików nie posiada
+katalogów - wszystkie Twoje pliki są po prostu przechowywane
+w jednym i tym samym miejscu.
+
+Język programowania Python zawiera łatwe w użyciu i bardzo skuteczne metody
+pracy z komputerowym systemem plików. W MicroPythonie na minikomputerze
+micro:bit zaimplementowano użyteczny podzbiór tych metod. Pozwalają one
+w łatwy sposób odczytywać i zapisywać pliki z i do urządzenia, jednocześnie
+zapewniając zgodność z innymi wersjami Pythona.
 
 .. warning::
 
-    Flashing your micro:bit will DESTROY ALL YOUR DATA since it re-writes all
-    the flash memory used by the device and the file system is stored in the
-    flash memory.
+    Wgrywanie programów na minikomputer micro:bit KASUJE WSZYSTKIE DANE
+    ponieważ operacja ta nadpisuje pamięć flash używaną przez urządzenie,
+    a system plików także jest w niej przechowywany.
 
-    However, if you switch off your device the data will remain intact until
-    you either delete it or re-flash the device.
+    Jeżeli jedynie wyłączysz swoje urządzenie, dane pozostaną nienaruszone
+    do czasu, aż je usuniesz albo załadujesz na urządzenie nowy program.
 
-Open Sesame
-+++++++++++
+Sezamie otwórz się
+++++++++++++++++++
 
-Reading and writing a file on the file system is achieved by the ``open``
-function. Once a file is opened you can do stuff with it until you close it
-(analogous with the way we use paper files). It is essential you close a file
-so MicroPython knows you've finished with it.
+Odczyt i zapis plików w systemie plików wykonywany jest za pomocą funkcji
+``open``. Kiedy plik jest już otwarty możesz z nim robić rozmaite rzeczy, do
```

Comment:
(ang. otwórz)

Reply:
Trochę pogubiliśmy się w tym. W niektórych przetłumaczonych już miejscach jest na przykład (ang. otwórz) a w innych (ang. open) a jeszcze w innych (pol. otwórz). Trzeba to będzie jeszcze raz sprawdzić i ujednolicić.